config/routes.rb

```diff
@@ -1,9 +1,20 @@
 Rails.application.routes.draw do
   root 'users#index'
+
   resources :users, param: :name, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
-    resources :comments, only: [:index, :show, :create, :update, :destroy]
-resources :likes, only: [:create, :destroy]
+    resources :posts, only: [:index, :show] do
+      resources :comments, only: [:create, :update, :destroy]
+
+      # Add routes for liking and unliking a post
+      post 'like', to: 'likes#create'
+      delete 'unlike', to: 'likes#destroy'
+
```

Comment:
- [ ] It would be nice if you fix the like creation because i have clicked the like button and page refreshed but i don't see any like in the database :fearful: Kindly fix this issue and test it well :+1: to make your project follow the requirements :blush: